lib/widgets/users/add_nodeId_widget.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:chirp_nets/utils/text.dart';
+import 'package:flutter/material.dart';
+import 'package:chirp_nets/models/user.dart';
+import 'package:chirp_nets/providers/users.dart';
+
+class SetUserInfoWidget extends StatefulWidget {
```

Comment:
The file name doesnt match the class names. Not sure if that is by design or not. It would be nice to have matching classes and file names.